public-api/vx/note.php

```diff
@@ -32,9 +32,23 @@
 			json_EmitFatalError_BadRequest(null, $RESPONSE);
 		}
 
-		$RESPONSE['note'] = noteComplete_GetByNode($ids[0]);
+		$RESPONSE['note'] = noteFlags_Filter(noteComplete_GetByNode($ids[0]), userAuth_GetID());
 
-		break; //case 'get': //note/get
+		break; //case 'get': //note/get		
+	case 'getnote': //note/getnote
+		// Temporary "Get note by note ID" call. To be renamed when the renaming time comes.
+		json_ValidateHTTPMethod('GET');
+
+		$ids = explode('+', json_ArgGet(0));
+
+		// Limit number of notes
+		if ( count($ids) > 250 ) {
+			json_EmitFatalError_BadRequest("Too many notes", $RESPONSE);
+		}
+		
+		$RESPONSE['notes'] = noteFlags_Filter(noteComplete_Get($ids), userAuth_GetID());
```

Comment:
I generally prefer plural for things that are lists, but I think the site generally uses `note` even when the returned items are lists with potentially many notes

Reply:
That's true, the one other location that returns notes does return with `note` :P I'm not too worried, this path is going to be reworked.